packages/core/src/WorkflowExecute.ts

```diff
@@ -796,7 +796,7 @@ export class WorkflowExecute {
 		});
 		if (workflowIssues !== null) {
 			throw new WorkflowOperationError(
-				'The workflow has issues and can for that reason not be executed. Please fix them first.',
+				`The workflow with id ${workflow.id} has issues and can for that reason not be executed. Please fix them first.`,
```

Comment:
All dynamic metadata should go on the `context` property on error objects. Otherwise each instance of the error creates a separate entry in our error tracker. 
To change this you'd need to 
1. Either update `WorkflowOperationError` constructor to accept custom `context`, or replace the `throw` with creating the error instance, setting `.context` on it, and then throwing that instance.
2. Update the frontend to render data from `context` in the error popup.